Leanplum-SDK/Classes/Internal/Leanplum.m

```diff
@@ -1993,6 +1995,23 @@ + (void)trackGeofence:(NSString *)event withValue:(double)value andInfo:(NSStrin
     LP_END_TRY
 }
 
++ (NSString *)getEventNameFromGeofenceType:(LPGeofenceEventType)event {
```

Comment:
Just 1 thing.
Can we please move the entire Enum logic  to a separate file and expose an Api for getting the name similar to Android. You can place the file in 
"Internal" directory.
 Something like this.

/  EnumConstants.h


#ifndef EnumConstants_h
#define EnumConstants_h

typedef enum {
    LPEnterRegion,
    LPExitRegion
} LPGeofenceEventType;
#endif